ql/instruments/makevanillaswap.hpp

```diff
@@ -112,6 +112,8 @@ namespace QuantLib {
         ext::optional<BusinessDayConvention> paymentConvention_;
 
         ext::shared_ptr<PricingEngine> engine_;
+
+        friend class SwapRateHelper;
```

Comment:
This is added to enable access within `SwapRateHelper::SwapRateHelper(const MakeVanillaSwap& swapBuilder)` to `swapBuilder.forwardStart_`.